MetaMask/metamask-extension
# Add Mint Blockchain

## **Description**
Add Mint Blockchain Mainnet network icon support.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27474?quickstart=1)

## **Related issues**

Fixes:N/A

## **Manual testing steps**

1. Go to [Mint](https://www.mintchain.io/)
2.Click the Metamask icon to add network
3.Open Metamask extension to check the network and token icons

## **Screenshots/Recordings**

### **Before**
<img width="330" alt="image" src="https://github.com/user-attachments/assets/b4ccf951-d1a8-4f60-ac20-de306b7f08d2">


### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
